app/test/unit/ci-checks-test.ts

```diff
@@ -12,8 +14,8 @@ describe('getCheckRunsGroupedByActionWorkflowNameAndEvent', () => {
     ]
     const groups = getCheckRunsGroupedByActionWorkflowNameAndEvent(checkRuns)
     const groupNames = [...groups.keys()]
-    expect(groupNames.includes('test1')).toBe(true)
-    expect(groupNames.includes('test2')).toBe(true)
+    assert.equal(groupNames.includes('test1'), true)
```

Comment:
You could do this
```suggestion
    assert(groupNames.includes('test1'))
```

or this
```suggestion
    assert.ok(groupNames.includes('test1'))
```
for all changes in this file, right?